TombEngine/Game/effects/tomb4fx.h

```diff
@@ -3,6 +3,7 @@
 #include "Game/Lara/lara_struct.h"
 #include "Math/Math.h"
 #include "Renderer/RendererEnums.h"
+#include <vector>
```

Comment:
I am not sure if this include is needed?

Reply:
Needed because the compiler needs to see a declaration of the type so that line 224 is valid C++

Reply:
it is not. Will change it.

Reply:
as we cannot guarantee that it will always be included elsewhere I added it again.

Reply:
It's included by `framework.h`, so resolved by `tomb4fx.cpp`. You don't need it. ;)

Reply:
then I remove it again :-D